content/pop-recovery.md

```diff
@@ -63,9 +63,7 @@ If the existing install is encrypted, please see the [encrypted disk](#encrypted
 
 If the existing OS install needs to be repaired, the installer application should be closed. An app menu is located in the top-left of the screen with the name of the currently running application (in this case: "Install Pop!\_OS"). Click on the app menu and select `Quit`. Alternatively, you can use the installer app to select keyboard and language settings, and then click the `Try Demo Mode` button in the lower-left corner of the install page.
 
->**Note:** Do not choose any install or repair options, as this could result in data loss.
-
->**Note:** Also, once you have setup or accessed any of the filesystems, you will need to boot to the recovery environment anew to use any of the install options.
+>**Note:** Do not choose any install or repair options, as this could result in data loss. If you have accessed any filesystems and intend to install or refresh afterwards, the machine should be rebooted into a fresh recovery environment to clear any created mount points.
```

Comment:
Now it's back to being a single comment. I still think these are two separate points, each deserving of their own note.

Reply:
It breaks the CI from what I can tell but I could be wrong.

Reply:
If wanted formatting breaks the CI, then the CI rules should be adjusted.

That said, if we have two separate things to say, then maybe a regular paragraph or a bullet-point list would make more sense than two highlighted notes in a row (which dilutes the highlighting, anyway.)

Reply:
It looks like the Markdown linter is interpreting the two blockquotes as one large blockquote, so adding an additional `>` on the blank line between them might help?